### PR TITLE
test: add additional tests to `utils/compact-adjacency-matrix`

### PR DESCRIPTION
**Pull request by uchiha-vivek**
Resolves # .

## Description 
Test Cases for adjacency matrix

> What is the purpose of this pull request?

This pull request:

-   a Added more test cases for adjacency matrix.
-   b Checking whether the matrix is null, updation of matrix, and checking for edges between the vertices.
-   c They have been implemented in form of test cases.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves # 
-   fixes # adding more test-cases

## Questions

> Any questions for reviewers of this pull request?

Please review the features and suggest changes if required.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by uchiha-vivek** [comment]
Please review it.

File: lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js [hunk 1]
```diff
@@ -20,31 +20,94 @@
 
 // MODULES //
 
-var tape = require( 'tape' );
-var instanceOf = require( '@stdlib/assert/instance-of' );
-var CompactAdjacencyMatrix = require( './../lib' );
+var tape = require('tape');
```
Comment (kgryte) [inline lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js:23]: All of these style changes should not have been made. Please revert and ensure that you follow project conventions.

File: lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js [hunk 2]
```diff
@@ -20,31 +20,94 @@
...
+	mat = compactAdjacencyMatrix(4);
+	t.strictEqual(instanceOf(mat, CompactAdjacencyMatrix), true, 'returns an instance');
 	t.end();
 });
+
+// adjacency Matrix can be depicted in form of undirected graph representation,directed graph represnetation and weighted graph representation
+
+tape('initialize the matrix correctly', function test(t) {
+	var mat = new CompactAdjacencyMatrix(4);
+	t.deepEqual(mat.matrix, [
```
Comment (kgryte) [inline lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js:56]: @uchiha-vivek Did you run these tests?

Based on what you have here, it does not appear so. There is no `matrix` attribute, and these tests are not written correctly.

**Review by kgryte — CHANGES_REQUESTED** [review]
This PR needs significant changes in order to be merged. I encourage you to see the tests for other packages (e.g., [`@stdlib/array/complex64`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/complex64/test)) for what we expect in terms of rigor and level of detail.

**Pull request closed without merging**